defi/src/protocols/data3.ts

```diff
@@ -32991,7 +32991,7 @@ const data3: Protocol[] = [
   },
   {
     id: "4179",
-    name: "Fixes World",
+    name: "Fixes 𝔉rc20",
```

Comment:
can we use another letter and not that one. Not sure if will work

Reply:
Done